instagram/src/components/PostContainer/Post.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import './PostContainer.css';
+import PropTypes from 'prop-types';
+
+
+
+
+const Post = props => {
```

Comment:
Imagine getting rid of all of the `props.` down below by doing something as short as this with object destructuring:
```javascript
const Post = ({ addLike, data, id }) => {
  // ...
```